.github/workflows/my-ci-pipeline.yml

```diff
@@ -0,0 +1,17 @@
+name: Continuous Integration
+on: 
+  push:
+    paths-ignore:
+      - 'README.md'
+  pull_request:
+    paths-ignore:
+      - 'README.md'
+
+jobs:
+  build:
+    name: Build and test
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - run: docker build --target test --tag todo-app:test .
```

Comment:
I'm a fan of attaching names to most stages rather than having the raw commands displayed something like below, but this is more a matter of preference than strong recommendation!
```yml
    - name: Build the test image
       run: docker build --target test --tag todo-app:test .
```